tests/api/utils/test_scheduler.py

```diff
@@ -1089,13 +1089,16 @@ async def test_schedule_convert_from_old_credentials_to_new(
     _assert_schedule_secrets(scheduler, project, schedule_name, None, None)
 
 
+@unittest.mock.patch.object(Scheduler, "_store_schedule_secrets_using_auth_secret")
```

Comment:
I think you can add the return value here no?
```suggestion
@unittest.mock.patch.object(Scheduler, "_store_schedule_secrets_using_auth_secret", return_value="auth-secret")
```

Reply:
@alonmr TIL 👍